tests/generate.py

```diff
@@ -48,6 +48,9 @@ def myreplacement(ex):
         patch = patch.replace('diff commits', 'Diff commits')
         patch = patch.replace('diffs for', 'Diffs for')
 
+    if '044115544bf6' in f:
+        patch = patch.replace('--- target_task_set@d009f0738a', '___ target_task_set@d009f0738a')
+
```

Comment:
Why?

Reply:
because `whatthepatch` sees a triple minus... so the generated json has an entry for this wrong file.
So it helps wtp to skip that stuff.

Reply:
:+1:

Reply:
Can you add this as a comment?